### ISSUE TITLE
Forbid inappropriate nodes in `class` body

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#368]
# Rule request

## Thesis

Currently we can write anything we want in `ClassDef` body.

```python
class Test(object):
    if some_call():
        def area(self):
            return 1
```

Or

```python
class Test(object):
    with open('./setup.cfg') as file:
        def area(self):
            return file
```

This is just insane! 

Things that should be allowed inside a class:
1. methods
2. docstrings
3. attributes

That's it. All other nodes should be a violation.

## Reasoning

Classes definition should be crystal clear. No dirty hacks!
